### DISCUSSION TITLE
[New Pause function]: Indicator missing

### DISCUSSION THREAD

**Discussion by Woisek** [vladmandic/sdnext#1241] [Ideas]
Just tested the new pause function.
When you press pause, nothing seems to happen. Only when you look at the console (which is not visible via network usage, for example) and see the timer rising, you notice something.

Since this is not very UX friendly, I would suggest the following:
Color the pause button in a bit darker color, so that it is clear that it does something completely different.
When the button is pressed, it should change to a red, so you can see that it is activated. Maybe also show the general "pause" icon ( | | ).
At the same time, the timer should not continue to run, but stop as well, as a further indication that "pause" has been pressed.
When pressed again, everything should be reversed (button darker color, timer runs again).

**Comment by vladmandic** [comment]
yup, all valid. and none possible from server side. i need to do a javascript callbacks to do visual indicators. its coming.
except the timer running - that's under gradio control, nothing i can do about that.

**Reply by vladmandic** [reply]
check now.

**Reply by Woisek** [reply]
Works. 👍

**Comment by Woisek** [comment]
Just a note: Is it possible to display again minutes too, instead only seconds?

**Reply by vladmandic** [reply]
done, i'll push the update in ~1h

**Reply by Woisek** [reply]
Perfect, works well. 👍
